### ISSUE TITLE
Failing e2e b2b\accessibility\tabbing-order.e2e-spec.ts

### ISSUE THREAD

**Issue by Platonn** [SAP/spartacus#10064]
1. run `yarn e2e:cy:open:b2b`
2. in the cypress menu run `b2b\accessibility\tabbing-order.e2e-spec.ts`

![obraz](https://user-images.githubusercontent.com/4001059/100467329-8fdfb400-30d2-11eb-9738-7a7582af388e.png)

**Comment by LTiger14** [comment]
Test was broken because of change in token storage.

I deleted the tab test and moved the behavior to the two main b2b checkout tests.

PR https://github.com/SAP/spartacus/pull/10072
